### PR TITLE
check `isLNURL` in `EnterPaymentInfoDialog`

### LINKED ISSUE #1164: Add lnurl support to the Paste Invoice or Address dialog in Breez mobile. It worked in the past
### How To Reproduce?

1. Go to Send -> Paste invoice or ID
2. Paste a lnurl
3. Touch approve
4. See the error instead of navigating to the next step

### Screenshot

Screenshot_20230615_082821_Breez

### Breez Version

0.15-beta Play Store

### PR DESCRIPTION
**Pull request by ubbabeck**
We did not check if the value of the payemnt is a lnurl for the `EnterPaymentInfoDialog`.

fixes #1164

### CODE REVIEW

File: lib/widgets/enter_payment_info_dialog.dart [hunk 1]
```diff
@@ -114,8 +114,11 @@ class EnterPaymentInfoDialogState extends State<EnterPaymentInfoDialog> {
                 validator: (value) {
                   if (parseNodeId(value) == null &&
                       _decodeInvoice(value) == null &&
-                      !isLightningAddress(value)) {
+                      !isLightningAddress(value) &&
+                      !isLNURL(value)) {
                     return texts.payment_info_dialog_error;
+                  } else if (isLNURL(value)) {
+                    widget.lnurlBloc.lnurlInputSink.add(value);
```
Comment (erdemyerebasmaz) [inline lib/widgets/enter_payment_info_dialog.dart:121]: This logic does not belong here, this functions only purpose is to validate if the input if valid or not.

The LNURL should be handled on pressing Approve.

Comment (ubbabeck) [reply lib/widgets/enter_payment_info_dialog.dart]: Agree, moved this logic. Please have a look at the latest rebase.

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

**Comment by ubbabeck** [comment]
## Tests for regressions
- [ ] nodeID & nodeURI
- [ ] lighting invoice
- [ ] lightning address LNURLP

**Pull request merged**